backend/src/models/actorModel.js

```diff
@@ -37,12 +42,62 @@ const ActorSchema = new Schema(
       enum: Object.values(BasicState),
       default: BasicState.ACTIVE
     },
-    createdAt: Number,
-    updatedAt: Number
+    customToken: {
```

Comment:
should we remove these 2? we don't need to persist them as they get renew everytime you login.